### DISCUSSION TITLE
AI-Mask conversion

### DISCUSSION THREAD

**Discussion by GeoGBo** [wkentaro/labelme#1406] [Q&A / Help]
Dear  all,
I just started working with labelme. I really like it so much.
Unfortunately, I started creating AI-Masks instead of AI-Polygons. In the JSON file, AI-Mask coordinates do not appear as plain text. Is there a possibility to convert these mask shape types into polygon shape types, where coordinates are available as plain text?
Best regards and many thanks in advance,
GeoGBO

**Comment by wkentaro** [comment]
@GeoGBo Do you want to get an image mask from shape_type="mask" annotation?
If so, this Twitter thread https://x.com/labelmeai/status/1737781486596833502 would help.

If you want to convert shape_type="mask" to "polygon", you can use this function:
https://github.com/labelmeai/labelme/blob/3a1f6554072dbf6e5b4a2cf6f5130399b5a708d5/labelme/ai/_utils.py#L14-L38

**Reply by GeoGBo** [reply]
Great, I followed the Twitter thread, it worked! Thank you so much! 👍
